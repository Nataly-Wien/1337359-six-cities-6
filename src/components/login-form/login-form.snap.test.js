import React from 'react';
import {Provider} from 'react-redux';
import {render} from '@testing-library/react';
import configureStore from 'redux-mock-store';
import LoginForm from './login-form';

const mockStore = configureStore({});
const store = mockStore({
  USER: {isLoginError: false}
});

it(`Should login form render correctly`, () => {
  const signIn = () => { };
  const {container} = render(<Provider store={store}>
    <LoginForm signIn={signIn} />
  </Provider>);

  expect(container).toMatchSnapshot();
});
