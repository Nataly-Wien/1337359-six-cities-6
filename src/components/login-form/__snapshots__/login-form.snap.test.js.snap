// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should login form render correctly 1`] = `
<div>
  <form
    action="#"
    class="login__form form"
    method="post"
  >
    <div
      class="login__input-wrapper form__input-wrapper"
    >
      <label
        class="visually-hidden"
      >
        E-mail
      </label>
      <input
        class="login__input form__input"
        name="email"
        placeholder="Email"
        type="email"
        value=""
      />
    </div>
    <div
      class="login__input-wrapper form__input-wrapper"
    >
      <label
        class="visually-hidden"
      >
        Password
      </label>
      <input
        class="login__input form__input"
        name="password"
        placeholder="Password"
        type="password"
        value=""
      />
    </div>
    <button
      class="login__submit form__submit button"
      type="submit"
    >
      Sign in
    </button>
  </form>
</div>
`;
